rebar.config

```diff
@@ -40,6 +40,11 @@
     {nodefinder, "2.0.7"}
 ]}.
 
+{dist_node, [
+    {setcookie, 'erldns-cookie'},
```

Comment:
Is this config in use anywhere here?

Reply:
Yes, by the tests, this config gives a cookie and a name inherited by all the rebar command, so that then tests can create new BEAM nodes and tests things in distribution. The `dnstest` tests are doing this, starting a new BEAM node with `erldns` running, makes tests cleaner by not polluting the node running them with state and apps being started.

Reply:
I'm somehow interpreted it related to the "other" cookie story 😆